backend/dal/Configuration/ContactConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Configuration
+{
+    /// <summary>
+    /// ContactConfiguration class, provides a way to configure the contact view in the database.
+    ///</summary>
+    public class ContactConfiguration : IEntityTypeConfiguration<Contact>
+    {
+        #region Methods
+        public void Configure(EntityTypeBuilder<Contact> builder)
+        {
+            builder.ToView("PIMS_CONTACT_VW");
```

Comment:
A contact is actually a view or an organization. This view is a union of both objects which allows easy sorting and filtering compared to using projections.

Reply:
The person/organization comment is in a few areas of the code base already, but I'll make sure it is here as well.